back-end/ProjectPlatinumShakeWebAPI/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
You should move these to the `.gitignore` file at the root of the project and modify them accordingly if needed.

Reply:
Done 👍